pyodbc.pyi,sha256=JWNpdgmrxhNioTjW6OoAEcXELcGZobcMYzKtYlcJRuY,32320
pyodbc.cpython-310-aarch64-linux-gnu.so,sha256=k_ZR3TtJ3BJzEY9AXgKeIk1KQ46h5XZlNfKypDZqwmI,1055056
pyodbc-4.0.39.dist-info/WHEEL,sha256=5FPtA1B6n-DIbcPbXnHVzp-as0KLsaNL-sSVkjvwO_0,154
pyodbc-4.0.39.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.39.dist-info/RECORD,,
pyodbc-4.0.39.dist-info/METADATA,sha256=Cu1OPw2btz_Dj3huRmD0y9-0CLuocCtBc9rxYLBxYyc,3160
pyodbc-4.0.39.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
